### PR TITLE
Add inlay hints to docs

### PR DESCRIPTION
**Pull request by predragnikolic**

### CODE REVIEW

**Comment by predragnikolic** [comment]
Here is how it looks:
Screenshot 2022-08-23 at 11 23 00

File: docs/src/features.md [hunk 1]
```diff
@@ -142,6 +142,21 @@ This package presents Code Lenses as a greenish clickable annotation positioned
...
+Inlay hints are short textual annotations that show parameter names, type hints.
+
+![inlay-hints](./images/inlay-hints.png)
+
+To enable inlay hints, modify the following settings through `Preferences: LSP Settings`:
+```js
+{
+    "show_inlay_hints": true
+}
+```
```
Comment (rchl) [inline docs/src/features.md:156]: I suppose we can just mention which option to enable. Including a whole code block with just that option sounds a bit excessive.

```suggestion
To enable inlay hints, enable the `show_inlay_hints` setting through `Preferences: LSP Settings`.
```

File: docs/src/features.md [hunk 2]
```diff
@@ -142,6 +142,21 @@ This package presents Code Lenses as a greenish clickable annotation positioned
...
+
+!!! info "Some LSP-* plugins require additional settings to be enabled in order to show inlay hints."
```
Comment (rchl) [inline docs/src/features.md:158]:
```suggestion
!!! info "Some servers require additional settings to be enabled in order to show inlay hints."
```

File: docs/src/features.md [hunk 3]
```diff
@@ -142,6 +142,21 @@ This package presents Code Lenses as a greenish clickable annotation positioned
 
 Sublime Text has no concept of Code Lenses.
 
+## Inlay Hints
+
+Inlay hints are short textual annotations that show parameter names, type hints.
```
Comment (rchl) [inline docs/src/features.md:147]: Maybe clarify that server needs to support it.

```suggestion
Inlay hints are short textual annotations that show parameter names and type hints for servers that support that feature.
```

**Review by rchl — APPROVED** [review]

**Pull request merged**